doc/examples/EEG_Example.md

```diff
@@ -0,0 +1,220 @@
+
+def skew_signal(t, m, e):
+    return scipy.stats.skew(m)
+```
+
+Now we'll pass the desired feature functions as a dictionary via the `custom_functions`
+keyword argument.
+
+```python
+guo_features = {
+    'mean': mean_signal,
+    'std': std_signal,
+    'mean2': mean_square_signal,
+    'abs_diffs': abs_diffs_signal,
+    'skew': skew_signal
+}
+
+fset_guo = featurize.featurize_time_series(eeg["times"], eeg["measurements"], None,
+                                           list(guo_features.keys()), eeg["classes"],
+                                           custom_functions=guo_features, use_celery=True)
+print(fset_guo)
+```
+
+### Multi-channel time series
+The EEG time series considered here consist of univariate signal measurements along a
+uniform time grid. But
+[`featurize_time_series`](https://mltsp.readthedocs.org/en/latest/api/mltsp.featurize.html#mltsp.featurize.featurize_time_series)
+also accepts multi-channel data; to demonstrate this, we will decompose each signal into
+five frequency bands using a discrete wavelet transform as suggested by [Subasi
+(2005)](http://www.sciencedirect.com/science/article/pii/S0957417404001745), and then
+featurize each band separately using the five functions from above.
+
+```python
+import pywt
+
+n_channels = 5
+eeg["dwts"] = [pywt.wavedec(m, pywt.Wavelet('db1'), level=n_channels-1)
+               for m in eeg["measurements"]]
+fset_dwt = featurize.featurize_time_series(None, eeg["dwts"], None,
+                                           list(guo_features.keys()), eeg["classes"],
+                                           custom_functions=guo_features)
+print(fset_dwt)
+```
+
+The output featureset has the same form as before, except now the `channel` coordinate is
+used to index the features by the corresponding frequency band. The functions in
+[`mltsp.build_model`](https://mltsp.readthedocs.org/en/latest/api/mltsp.build_model.html)
+and [`mltsp.predict`](https://mltsp.readthedocs.org/en/latest/api/mltsp.predict.html)
+all accept featuresets from single- or multi-channel data, so no additional steps are
+required to train models or make predictions for multichannel featuresets using the
+`mltsp` library.
+
+## Model Building
+Model building in `mltsp` is handled by the
+[`build_model_from_featureset`](https://mltsp.readthedocs.org/en/latest/api/mltsp.build_model.html#mltsp.build_model.build_model_from_featureset)
+function in the `mltsp.build_model` submodule. The featureset output by
+[`featurize_time_series`](https://mltsp.readthedocs.org/en/latest/api/mltsp.featurize.html#mltsp.featurize.featurize_time_series)
+contains both the feature and target information needed to train a
+model; `build_model_from_featureset` is simply a wrapper that calls the `fit` method of a
+given `scikit-learn` model with the appropriate inputs. In the case of multichannel
+features, it also handles reshaping the featureset into a (rectangular) form that is
+compatible with `scikit-learn`.
```

Comment:
`build_model.rectangularize_featureset`; it's not something the user should worry about so I didn't mention it. Internally it calls `featureset.to_dataframe().unstack(level='channel')`, which unravels the (name, channel) multi-index into a single univariate index.